flink-connector-redis/pom.xml

```diff
@@ -34,7 +34,7 @@ under the License.
     <packaging>jar</packaging>
 
     <properties>
-        <jedis.version>2.9.0</jedis.version>
+        <jedis.version>3.7.1</jedis.version>
```

Comment:
is this mandatory for flink update?
if not will be better in other commit, only with this update (and related)